src/main/java/com/suse/saltstack/netapi/calls/modules/Match.java

```diff
@@ -0,0 +1,61 @@
+package com.suse.saltstack.netapi.calls.modules;
+
+import com.google.gson.reflect.TypeToken;
+import com.suse.saltstack.netapi.calls.LocalCall;
+import com.suse.saltstack.netapi.utils.ClientUtils;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * salt.modules.match
+ */
+public class Match {
+
+    public static LocalCall<Boolean> compound(String tgt, Optional<String> minionId) {
```

Comment:
Even if we provide documentation it would be mostly copied from the salt docs and would not be precise since our library may work with different versions of the module which could have slightly different but documented behavior.